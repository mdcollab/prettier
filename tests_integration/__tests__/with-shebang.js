"use strict";

const runPrettier = require("../runPrettier");

test("preserves shebang", () => {
  const result = runPrettier("cli/with-shebang", ["issue1890.js"]);

  expect(result.stdout).toMatchSnapshot();
  expect(result.status).toEqual(0);
});
