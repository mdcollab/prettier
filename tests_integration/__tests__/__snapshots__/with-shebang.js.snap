// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`preserves shebang 1`] = `
"#!/usr/bin/env node
\\"use strict\\";
"
`;
